airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -545,17 +544,17 @@ def monitor_launched_pod(self, launcher, pod) -> Tuple[State, Optional[str]]:
         :return:
         """
         try:
-            (final_state, result) = launcher.monitor_pod(pod, get_logs=self.get_logs)
+            (final_state, remote_pod, result) = launcher.monitor_pod(pod, get_logs=self.get_logs)
         finally:
             if self.is_delete_operator_pod:
                 launcher.delete_pod(pod)
         if final_state != State.SUCCESS:
             if self.log_events_on_failure:
-                for event in launcher.read_pod_events(pod).items:
+                for event in launcher.read_pod_events(self.pod).items:
```

Comment:
```suggestion
                for event in launcher.read_pod_events(pod).items:
```
Since we take a pod as an arg, lets use that instead just so it behaves as one would expect, even though right now its the same thing.

Reply:
Works for me.  Do we want to update the one on line 555 to match too then? It's `self.pod` in main

Reply:
Yeah, good catch, missed that one.

Reply:
👍 Fixed up.  Hopefully no weird CI failures this time.